modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -0,0 +1,261 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019 Intel Corporation
+
+#include "../test_precomp.hpp"
+
+#ifdef HAVE_INF_ENGINE
+
+#include <stdexcept>
+
+#include <inference_engine.hpp>
```

Comment:
Please guard this with [deprecation warning suppression](https://github.com/opencv/opencv/pull/15139/files#diff-498d277c8c35051874b40e5e8167ccf2R19) for OpenVINO 2019R2.

Reply:
Done, thanks!